core/src/main/java/org/aerogear/mobile/core/Request.java

```diff
@@ -0,0 +1,61 @@
+package org.aerogear.mobile.core;
```

Comment:
Should we move this class into the reactive package since this is reactive related?